practice-app/api/tests.py

```diff
@@ -1,3 +1,15 @@
-from django.test import TestCase
-
-# Create your tests here.
+from django.test import TestCase, Client
+
+
+class GettingPlacesTestCase(TestCase):
+    def test_get_places(self):

```

Comment:
I think it would be better if you seperate the assertions from each other by using multiple test methods to avoid complications when you get an error from the tests. 
You can maybe use 3 methods for 3 responses.